docker-compose.build.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+services:
+    gradle:
+        build:
+            context: .
+            dockerfile: Dockerfile-gradle-with-tc
+        command: >
+            sh -c "cd /Octopus &&
+            ./gradlew -Dteamcity.distribution=/teamcity clean distZip &&
+            chmod 777 /Octopus/build/distributions/* && 
+            cp /Octopus/build/distributions/*.zip /Octopus/docker_data/teamcity-server/datadir/plugins"
```

Comment:
nit: this line is a bit of a hack, as it effectively has knowledge of the other teamcity docker-compose file structure.
This works nicely, but there's other ways to get the plugin into TC.

Reply:
As noted above, this is just due to lack of in-depth TC knowledge :) ill see if i can simplify it based on what you described in the other comment,